custom/conf/app.ini.sample

```diff
@@ -290,7 +290,7 @@ ENABLE_GZIP = false
 ENABLE_PPROF = false
 ; PPROF_DATA_PATH, use an absolute path when you start gitea as service
 PPROF_DATA_PATH = data/tmp/pprof
-; Landing page, can be "home", "explore", or "organizations"
+; Landing page, can be "home", "explore", "organizations" or "login"
```

Comment:
Maybe add a line that indicate that the login option is not a security measure. It is just change the ui flow and that it may be wise to set the option REQUIRE_SIGNIN_VIEW à true if needed.
```suggestion
; Landing page, can be "home", "explore", "organizations" or "login"
; The "login" choice is not a security measure but just an ui flow change and you may need to look at REQUIRE_SIGNIN_VIEW option for security concern.
```